### ISSUE TITLE
Errors when glance card inside vertical-stack-in-card [HA 0.80.1]

### ISSUE THREAD

**Issue by bieniu** [custom-cards/vertical-stack-in-card#26]
After update HA to 0.80.1 I have a lot errors in log:
```
2018-10-15 21:29:04 ERROR (MainThread) [frontend.js.latest.201810140] https://__redacted__/local/lovelace/vertical-stack-in-card.js?v=0.0.6:58:45 Uncaught TypeError: Cannot read property 'childNodes' of null
```
This happens only when I have glance card inside verical-stack-in-card.

Example config:
```
          - type: custom:vertical-stack-in-card
            cards:
              - type: glance
                title: 'Salon'
                show_name: false
                column_width: calc(100% / 4)
                entities:
                  - sensor.temperature_salon
                  - sensor.humidity_salon
                  - sensor.co2_salon
                  - sensor.illumination_salon
              - type: entities
                show_header_toggle: false
                entities:
                  - entity: binary_sensor.motion_salon
                    secondary_info: last-changed
                  - climate.thermostat_salon
                  - switch.light_bulbs_salon
                  - switch.light_leds_salon
```
Beetwen glance and entities card there is a line. On older versions of HA there wasn't any line.

![image](https://user-images.githubusercontent.com/478555/46976038-95333d80-d0c8-11e8-89c4-79d9a374a95a.png)

**Comment by SilvrrGIT** [comment]
Getting the same error.  

Config is here if needed: https://github.com/SilvrrGIT/HomeAssistant/blob/master/ui-lovelace.yaml

**Comment by cicero200272** [comment]
Confirmed, same issue here (glance card within vertical-stack-in-card). It is working with other card types such as entities card and picture entity card.

**Comment by Boosik** [comment]
Same problem, really serious because entire lovelace-ui breaks.

**Comment by ofekashery** [comment]
Fixed in new version([0.0.7](https://github.com/custom-cards/vertical-stack-in-card/releases/tag/v0.0.7)), Thanks @thomasloven!
